tce/src/applibs/PIG/AsciiImageWriter.hh

```diff
@@ -34,6 +34,9 @@
 #ifndef TTA_ASCII_IMAGE_WRITER_HH
 #define TTA_ASCII_IMAGE_WRITER_HH
 
+#include <cstdint>
```

Comment:
If not needed in the header, please move the includes to the .cc to reduce cpp effort.